src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+unreleased
+++++++++++++++++++
+* Add examples for commands.
```

Comment:
History entries aren't required or desired for help text changes. Only customer facing bug fixes and features.